contracts/instruments/RibbonVolatility.sol

```diff
@@ -352,7 +372,8 @@ contract RibbonVolatility is DSMath, InstrumentStorageV1, InstrumentStorageV2 {
                 collateralAsset,
                 expiry,
                 strikePrice,
-                optionType
+                optionType,
+                paymentToken
```

Comment:
Referring to this [comment](https://github.com/ribbon-finance/structured-products/pull/58/files?file-filters%5B%5D=.sol#r581278291), we can use `address(0)` instead of passing the `paymentToken`.

Reply:
done